src/main/java/org/broadinstitute/hellbender/engine/filters/CommandLineFilter.java

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import java.io.Serializable;
+
+/**
+ * Interface implemented by command-line accessible read filters for argument validation.
+ */
+public interface CommandLineFilter extends Serializable {
```

Comment:
Agreed - it evolved from its original purpose. I think maybe "ValidatingFilter" since it basically adds a validate method, and its not really read filter specific.